### ISSUE TITLE
is this package still being maintained?

### ISSUE THREAD

**Issue by mkoohafkan** [Zelazny7/isofor#24]
As the title says---is this package still being maintained, and are there any plans to publish to CRAN? I am considering using this package in my workflow but will seek out an alternative if no release is planned. Thanks!

**Comment by gravesee** [comment]
Yes, it is casually maintained. I can work on publishing it to CRAN once I prune some of the work-specific features.

**Comment by gravesee** [comment]
just submitted to CRAN

**Comment by gravesee** [comment]
It is considered feature complete. And it is still maintained casually. I
welcome contributions for submitting to CRAN.

If I have time this weekend I can create a CRAN branch and prune some of
the work specific features I've added to make it more likely to pass the
checks.
